streamdeck/com.udts.sdPlugin/manifest.json

```
@@ -38,7 +45,8 @@
       "Name": "Force Refresh Stream",
       "States": [
         {
-          "Image": "action/images/forcerefreshstream"
+          "Image": "action/images/forcerefreshstream",
+          "Title": "Force\nRefresh\nStream"
```

based on my back of napkin testing this might look better if it just says Refresh Stream. we should change the button in the app to do that as well, and add it to the web controller 
```suggestion
          "Title": "Refresh\nStream"
```